scripts/startup.sh

```diff
@@ -297,36 +351,24 @@ installtype () {
   set_option INSTALL_TYPE $install_type
 }
 
-# More features in future
+# === More features in future ===
 # language (){}
 
-# Starting functions
-clear
-logo
+# === setup basic options for server installation ===
 userinfo
-clear
-logo
+grubOptions
 desktopenv
-# Set fixed options that installation uses if user choses server installation
+
 set_option INSTALL_TYPE MINIMAL
-set_option AUR_HELPER NONE
```

Comment:
I've hard-coded this line for this, on line 50 of your modification of 2-user.sh, you have a line stating:
if [[ ! $AUR_HELPER == none ]]; then

If user chooses server installation, it will not ask user for aur helper, thus, $AUR_HELPER will not be set to none. This is why this line is here for, so that it will put it as NONE. It will still be re-written to user selected AUR if installation is not server.
Maybe I missed your edit where you got it working without this, but I couldn't find it...